pkg/console/operator/operator.go

```diff
@@ -217,17 +217,15 @@ func (c *consoleOperator) handleSync(configs configSet) error {
 		return fmt.Errorf("console is in an unknown state: %v", updatedStatus.Spec.ManagementState)
 	}
 
-	// set default conditions ok first, sync can toggle if conditions are invalid
-	c.ConditionsDefault(updatedStatus)
 	err := c.sync_v400(updatedStatus, configs)
 	if err != nil {
-		if !customerrors.IsSyncError(err) || !customerrors.IsCustomLogoError(err) {
```

Comment:
Here the condition felt to be vice versa...  @benjaminapetersen can you confirm ?